text_extensions_for_pandas/io/test_conll.py

```diff
@@ -731,6 +732,43 @@ def test_compute_accuracy(self):
             ),
         )
 
+    def test_maybe_download_dataset(self):
+        base_dir = "test_data/io/test_conll"
+        ewt_dir = base_dir + "/ewt"
+        conll9_dir = base_dir + "/conll9"
+        ewt_url = "https://github.com/UniversalDependencies/UD_English-EWT/blob/master/en_ewt-ud-dev.conllu"
+        conll_09_test_data_url = (
+            "https://ufal.mff.cuni.cz/conll2009-st/trial/CoNLL2009-ST-English-trial.zip"
+        )
+
+        # test download of file
+        val = maybe_download_dataset_data(ewt_dir, ewt_url)
+        self.assertEqual(val, ewt_dir + "/en_ewt-ud-dev.conllu")
+        self.assertTrue(os.path.isdir(ewt_dir))
+        self.assertTrue(os.path.isfile(ewt_dir + "/en_ewt-ud-dev.conllu"))
+        # test download of
+        val = maybe_download_dataset_data(ewt_dir, ewt_url, alternate_name="dev.conllu")
+        self.assertEqual(val, ewt_dir + "/dev.conllu")
+        self.assertTrue(os.path.isdir(ewt_dir))
+        self.assertTrue(os.path.isfile(ewt_dir + "/dev.conllu"))
+        # check we didn't overwrite the last file
+        self.assertTrue(os.path.isfile(ewt_dir + "/en_ewt-ud-dev.conllu"))
+
```

Comment:
You should call `maybe_download_dataset_data()` again here to make sure that the function works if the local file exists.

Reply:
Fixed in latest commit